tsconfig-aot.json

```diff
@@ -13,7 +13,12 @@
     "noEmitHelpers": true,
     "rootDir": ".",
     "noUnusedLocals": true,
-    "noUnusedParameters": true
+    "noUnusedParameters": true,
+    "types": [
```

Comment:
Where I got the info from if useful: http://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types